# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/Layers.h
# Commit: c09a51622e98
# Full Hash: c09a51622e988e81077c75e3aaa5a49df66284b3
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-22 16:44:08
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 737807563dd5 -r c09a51622e98 gfx/layers/Layers.h
--- a/gfx/layers/Layers.h	Thu Mar 21 23:14:26 2019 +0000
+++ b/gfx/layers/Layers.h	Fri Mar 22 03:15:14 2019 +0000
@@ -791,13 +791,14 @@
    */
   virtual bool SetPendingScrollUpdateForNextTransaction(
       ScrollableLayerGuid::ViewID aScrollId,
-      const ScrollUpdateInfo& aUpdateInfo);
+      const ScrollUpdateInfo& aUpdateInfo,
+      wr::RenderRoot aRenderRoot);
   Maybe<ScrollUpdateInfo> GetPendingScrollInfoUpdate(
       ScrollableLayerGuid::ViewID aScrollId);
   void ClearPendingScrollInfoUpdate();
 
  protected:
-  ScrollUpdatesMap mPendingScrollUpdates;
+  wr::RenderRootArray<ScrollUpdatesMap> mPendingScrollUpdates;
 };
 
 /**